x/ibc/light-clients/07-tendermint/client/cli/tx.go

```diff
@@ -34,31 +34,29 @@ const (
 // in https://github.com/cosmos/ics/tree/master/spec/ics-002-client-semantics#create
 func NewCreateClientCmd() *cobra.Command {
```

Comment:
Note a TODO on these commands. They should return the generated identifier to the user

Reply:
I'm just going to add to the issue that removes all these commands